proto/src/main/java/com/google/gson/protobuf/ProtoTypeAdapter.java

```diff
@@ -190,6 +191,9 @@ public static Builder newBuilder() {
   private static final com.google.protobuf.Descriptors.FieldDescriptor.Type ENUM_TYPE =
       com.google.protobuf.Descriptors.FieldDescriptor.Type.ENUM;
 
+  private static final ConcurrentMap<String, Map<Class<?>, Method>> mapOfMapOfMethods =
+      new MapMaker().makeMap();
```

Comment:
Oh, you're right that this is static. Ugh. I think the fix we want to ship is one where we switch this to `TypeAdapterFactory`, where we don't need any maps at all.